conversion/conversion.go

```diff
@@ -429,13 +429,8 @@ func CreateDatabase(project, instance, dbName string, conv *internal.Conv, out *
 
 // UpdateDDLForeignKeys updates the Spanner database with foreign key
 // constraints using ALTER TABLE statements.
-func UpdateDDLForeignKeys(project, instance, dbName string, conv *internal.Conv, out *os.File) error {
+func UpdateDDLForeignKeys(project, instance, dbName string, maxWorkers int64, conv *internal.Conv, out *os.File) error {
```

Comment:
as discussed in the other comment, I think making this a top-level flag is an overkill ~ a simpler option would be to just make it a variable in this (conversion) package

Reply:
Retaining this to set various workers for unit tests. Declaring the variable globally in cmd.go instead from where this function is called.